src/py/flwr/cli/run/run.py

```diff
@@ -62,6 +68,13 @@ def run(
             "inside the `pyproject.toml` in order to be properly overriden.",
         ),
     ] = None,
+    stream: Annotated[
+        bool,
+        typer.Option(
+            "--stream/--show",
```

Comment:
Should we use `--log` here (true when set, false otherwise)? `--show` seems misleading b/c nothing would be shown

Reply:
Good point. You're right that `--show` isn't intuitive here. But to me, I read the word `log` more naturally as a noun instead of a verb. Maybe this should just be `--stream` and is true when set, false otherwise. Then the help text is "Use `--stream` with `flwr run` to display logs; omit to suppress them".

Reply:
We can start with that. Just maybe not the "suppress" part since it's technically wrong.